File: Python/Disk.py
# kelas yang digunakan untuk mengimplementasikan Disk
# kelas Disk
class Disk:
    # private atribute dari kelas Disk
    __type = ""
    __capacity = ""
    __price = 0

    def __init__(self, type="", capacity="", price=0):
        # konstruktor
        self.__type = type
        self.__capacity = capacity
        self.__price = price

    # mengeset nilai atribut type
    def setType(self, type):
        self.__type = type

    # mengembalikan nilai atribut type
    def getType(self):
        return self.__type

    # mengeset nilai atribut capacity
    def setCapacity(self, capacity):
        self.__capacity = capacity

    # mengembalikan nilai atribut capacity
    def getCapacity(self):
        return self.__capacity

    # mengeset nilai atribut price
    def setPrice(self, price):
        self.__price = price

    # mengembalikan nilai atribut price
    def getPrice(self):
        return self.__price

    # menampilkan atribut Disk
    def printDisk(self):
        print("Type : " + self.getType())
        print("Capacity : " + self.getCapacity())
        print("Price : " + str(self.getPrice()))

File: PHP/Disk.php
<?php
// kelas yang digunakan untuk mengimplementasikan Disk
class Disk
{
    // private atribute dari kelas Disk
    private $type = "";
    private $capacity = "";
    private $price = 0;

    function __construct($type = "", $capacity = "", $price = 0)
    {
        // konstruktor
        $this->type = $type;
        $this->capacity = $capacity;
        $this->price = $price;
    }
    // mengeset nilai atribut type
    function setType($type)
    {
        $this->type = $type;
    }
    // mengembalikan nilai atribut type
    function getType()
    {
        return $this->type;
    }
    // mengeset nilai atribut capacity
    function setCapacity($capacity)
    {
        $this->capacity = $capacity;
    }
    // mengembalikan nilai atribut capacity
    function getCapacity()
    {
        return $this->capacity;
    }
    // mengeset nilai atribut price
    function setPrice($price)
    {
        $this->price = $price;
    }
    // mengembalikan nilai atribut price
    function getPrice()
    {
        return $this->price;
    }
    // menampilkan atribut Disk
    function printDisk()
    {
        echo "Type : " . $this->getType() . "<br>";
        echo "Capacity : " . $this->getCapacity() . "<br>";
        echo "Price : " . $this->getPrice() . "<br>";
    }

    function __destruct()
    {
    }
}
